pypeman/helpers/serializers.py

```diff
@@ -0,0 +1,30 @@
+"""
+serializers, that might be helpful for pypeman projects
+"""
+
+import json
+
+
+# attempt to create serializers allowing to encode objects for non
+# Python applications.
+class JsonableEncoder:
+    """ Initial simple encoder for objects.
+        returns either
+        - the object itself if it can be json encoded
+        - a slightly transformed object, that can be json encoded (To be implemented)
+        - a dict containing two fields: type and repr
+    """
+    def encode(self, obj):
+        typ = None
+        try:
+            rslt = json.dumps(obj)
+            typ = "json"
+            return jsonstr
```

Comment:
What is jsonstr ? Do you mean result ?

Reply:
yes, this is probably what I mean.

Perhaps we should add some automatic syntax checker like flake8 to the project.
We use it more and more for other projects.
If not writing a huge flake config file it can sometimes be a pain getting the code clean, but for most projects it's worth to just use an almost standard flake8 config and force the devs to make the code pass

Reply:
Yes better :D

Reply:
pls resolve all conversations that you consider being treated completely